### DISCUSSION TITLE
Stuck in the "meshing" with an orange bar, what to do?

### DISCUSSION THREAD

**Discussion by joaonoob22** [alicevision/Meshroom#2355] [General]
Hey guys i'm stuck in this since yesterday and I don't know what to do, any tips? Thank you!

![why-is-it-stuck-in-the-meshing-v0-yy3jz759marc1](https://github.com/alicevision/Meshroom/assets/165405300/a66245b3-0e30-4190-a233-a92c966d6417)

**Comment by natowi** [comment]
Try clicking on the three dots in the right corner -> clear pending status
